docs/integrations/destinations/local-json.md

```diff
@@ -49,7 +49,9 @@ The local mount is mounted by Docker onto `LOCAL_ROOT`. This means the `/local`
 If your Airbyte instance is running on the same computer that you are navigating with, you can open your browser and enter [file:///tmp/airbyte\_local](file:///tmp/airbyte_local) to look at the replicated data locally.
 If the first approach fails or if your Airbyte instance is running on a remote server, follow the following steps to access the replicated files:
 1. Access the scheduler container using `docker exec -it airbyte-scheduler bash`
-2. Navigate to the default local mount using `cd /tmp/airbyte_local`
-3. Navigate to the replicated file directory you specified when you created the destination, using `cd /{Directory_Specified}`
-4. List files containing the replicated data using `ls`
-5. Execute `cat {filename}` to display the data in a particular file
+1. Navigate to the default local mount using `cd /tmp/airbyte_local`
+1. Navigate to the replicated file directory you specified when you created the destination, using `cd /{Directory_Specified}`
+1. List files containing the replicated data using `ls`
+1. Execute `cat {filename}` to display the data in a particular file
+
+Note: Above steps is also applicable if you are running Airbyte on Windows OS with WSL2 and Docker.
```

Comment:
@ppatali maybe adding a link to this https://github.com/airbytehq/airbyte/blob/master/docs/tutorials/locating-files-local-destination.md when using Windows. WDYT?

Reply:
That makes sense. I will push a commit to add this link